ui/src/app/services/data/data.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable } from '@angular/core'
```

Comment:
Please update dist/ files before merging the PR

Reply:
Will handle before the Merge.